pkg/analytics/analytics.go

```diff
@@ -58,6 +58,7 @@ const (
 	signupEvent              = "Signup"
 	disableEvent             = "Disable Analytics"
 	stackNotSupportedField   = "Stack Field Not Supported"
+	stackIsComposeFile       = "Stack Is compose file"
```

Comment:
```suggestion
	stackIsComposeFile       = "Stack Is Compose File"
```

Reply:
cc @rlamana

Reply:
I would add an extra property to the `Stack Deploy` event